.github/workflows/android.yml

```diff
@@ -239,6 +239,7 @@ jobs:
               -DQT_HOST_PATH=$QT_BASE/macos \
               -DQT_ANDROID_SIGN_APK=Yes \
               -DQT_ANDROID_SIGN_AAB=Yes \
+              -DUSE_MM_SERVER_API_KEY=Yes \
```

Comment:
Based on `INSTALL.md`, this setting seems to be used only on Android and Linux

Reply:
Not sure what is in INSTALL , it may need update too. 
It is missing in ios.yml, macOS*.yml and win.yml too. e.g.: 
Also make sure it is used in all cmake invocations - there may be debug and release builds in each (so you need to add it 2 times)

https://github.com/MerginMaps/mobile/blob/524ee438ffea697381da5747a83022756b339554/.github/workflows/ios.yml#L196